protos/action/action.proto

```diff
@@ -177,6 +183,26 @@ message GotoLocationResponse {
     ActionResult action_result = 1;
 }
 
+message DoOrbitRequest {
+    enum OrbitYawBehavior {
+        ORBIT_YAW_BEHAVIOUR_HOLD_FRONT_TO_CIRCLE_CENTER = 0;
+        ORBIT_YAW_BEHAVIOUR_HOLD_INITIAL_HEADING = 1;
+        ORBIT_YAW_BEHAVIOUR_UNCONTROLLED = 2;
+        ORBIT_YAW_BEHAVIOUR_HOLD_FRONT_TANGENT_TO_CIRCLE = 3;
+        ORBIT_YAW_BEHAVIOUR_RC_CONTROLLED = 4;
+    }
```

Comment:
Nice! However, I think the code won't generate properly with the enum inside of `DoOrbitRequest` (did you try to generate the corresponding code?). The simplest fix would be to move the enum out of it (e.g. to line 185, or at the end of the file, after the *Request definitions).

So the fix is very easy: just move this enum out of this message :blush:.

Reply:
Cool, done.